quadlets/filebrowser/filebrowser.container

```diff
@@ -0,0 +1,25 @@
+[Unit]
+Description=Filebrowser Quadlet
+
+[Service]
+Restart=always
+TimeoutStartSec=900
+
+[Install]
+WantedBy=default.target
+
+[Container]
+Image=docker.io/filebrowser/filebrowser:latest
+ContainerName=filebrowser
+HostName=filebrowser
+
+# You will have to manually create the config and db file on first run as it mounts files not dirs
```

Comment:
I want to clarify what this is about and see if there's something we can do better about that.

There's another quadlet here that had a quirk similar to this. I need to check on which that was and revisit it also. One that stands out is `calibre-web` because it wants a `metadata.db` from an existing `calibre` library. I added a `README.md` for that situation, but I seem to remember another one that needed to be bootstrapped.

Reply:
I did try and add a post exec with a symlink but that failed I can look into this again though and try work out a solution.